src/routes/lib/v1/PackageRequest.js

```diff
@@ -150,6 +172,36 @@ class PackageRequest extends BaseRequest {
 		});
 	}
 
+	async getResolvedEntrypoints () {
```

Comment:
This still seems to consider JS and CSS in one step. E.g., when there's a "safe" entry for JS, and nothing for CSS, it doesn't continue to cdnjs/stats for CSS - it should continue until both JS and CSS are found.

Reply:
I updated the logic